src/main/java/org/ow2/proactive/catalog/service/BucketService.java

```diff
@@ -168,4 +170,19 @@ private BucketEntity findBucketByNameAndCheck(String bucketName) {
         }
         return bucketEntity;
     }
+
+    public List<BucketMetadata> getBucketsByGroups(String sessionId, String ownerName, Optional<String> kind,
+            Optional<String> contentType, Supplier<List<String>> authenticatedUserGroupsSupplier)
+            throws NotAuthenticatedException, AccessDeniedException {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the redundant '!unknownSymbol!' thrown exception declaration(s). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)